doc/en/user/source/data/cascaded/wms.rst

```diff
@@ -62,6 +62,8 @@ Connecting a remote WMS allows for the following features:
 
 * Full **REST Configuration**. See the :ref:`rest` section for more information about the GeoServer REST interface.
 
+* **Optimization**. It is possible to ignore remote WMS calls if requested map area is completely outside the native bounds as advertised by remote WMS layer. These settings are available under **Publishing** Tab.
+
```

Comment:
This documentation update was insufficient:

* REST API is documented using OpenAPI
* User interface changes to the publishing tab should be documented

Reply:
@jodygarnett not sure about the context of REST API.

for the optimization part, will surely add images to documentation

Reply:
The rest api stuff is [here](https://github.com/geoserver/geoserver/tree/master/doc/en/api/1.0.0) - specially this: https://docs.geoserver.org/latest/en/api/#/latest/en/api/1.0.0/wmsstores.yaml

Thanks for updating the docs!